### PR TITLE
[16.0][MIG] purchase_order_line_stock_available: Migration to 16.0

### PR DESCRIPTION
**Pull request by victoralmau**
Migration to 16.0

Please @carolinafernandez-tecnativa can you review it?

@Tecnativa TT49812

### CODE REVIEW

**Review by carolinafernandez-tecnativa — APPROVED** [review]
Functional review LGTM :) Thanks!

File: purchase_order_line_stock_available/views/purchase_view.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    <record id="purchase_order_form_view_stock" model="ir.ui.view">
+        <field name="name">purchase.order.form.view.stock</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.purchase_order_form" />
+        <field name="arch" type="xml">
+            <xpath
+                expr="//field[@name='order_line']/tree//field[@name='qty_invoiced']"
+                position="after"
+            >
+                <field name="virtual_available" />
```
Comment (carlos-lopez-tecnativa) [inline purchase_order_line_stock_available/views/purchase_view.xml]: Could you make the field optional? This way, the user can easily show or hide the field.
```suggestion
                <field name="virtual_available" optional="show"/>
```

**Review by carlos-lopez-tecnativa — APPROVED** [review]
LGTM

**Pull request merged**